### PR TITLE
WatchTower-Technology

### PR DESCRIPTION
**Pull request by WatchTower-Technology**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,3 @@
   <img src="https://github.com/microsoft.png" width="40px" height="40px" style="border-radius:50%"  />
   <a href="https://github.com/microsoft">Microsoft</a>
 </p>
-
-<p style="display: flex; align-items:center; gap:10px;">
-  <img src="https://github.com/SulaimonShittu.png" width="40px" height="40px" style="border-radius:50%"  />
-  <a href="https://github.com/SulaimonShittu">Sulaimon Shittu</a>
-</p>
```
Comment (boredmage) [inline README.md]: This PR is removing this user data, rather you should sync update with your forked version before making a PR

**Review by boredmage — CHANGES_REQUESTED** [review]
You should fix that

**Comment by WatchTower-Technology** [comment]
I don’t still understand

**Comment by DevMarkson** [comment]
I would advice you delete this pull request and the fork, re fork the repo then add yourself as a participant again in your branch

**Pull request merged**